client/static/images/stuff2.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="256" height="256" viewBox="0 0 256 256" xml:space="preserve">
```

Comment:
can we please give this file a more descriptive name

Reply:
oh yeah sorry about that lol, resolved